tasks/get_current_archive_list/get_current_archive_list.py

```diff
@@ -427,7 +427,14 @@ def translate_s3_import_to_partitioned_data_sql() -> TextClause:
     return text(
         f"""
         INSERT INTO orca.reconcile_s3_object (job_id, orca_archive_location, key_path, etag, last_update, size_in_bytes, storage_class, delete_marker)
-            SELECT :job_id, orca_archive_location, key_path, etag, last_update, size_in_bytes, storage_class, delete_marker
+            SELECT 
+            :job_id, 
+            orca_archive_location, 
+            key_path, 
+            CONCAT('"', etag, '"') as etag, /* copy_to_glacier's AWS call presently wraps this in quotes. Seems like a bug, but is shown on https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.list_object_versions */
+            last_update, 
+            CAST(size_in_bytes AS BIGINT), 
```

Comment:
See notes on keeping this as bigint type instead of text. In either instance, you should probably use COALESCE (see https://www.postgresql.org/docs/10/functions-conditional.html#FUNCTIONS-COALESCE-NVL-IFNULL)  like the following `COALESCE(size_in_bytes, 0)` or if you keep it text `COALESCE(CAST(size_in_bytes AS BIGINT, 0)` Since  the table you insert to requires that `size_in_bytes be non-null (see https://github.com/nasa/cumulus-orca/blob/10dedf86495a6a69fca7c044279ab66d19fef0b8/tasks/db_deploy/install/orca_sql.py#L780) This guarantees any NULL values AWS throws us will show up as 0 and not cause us issues with our current schema.